src/policy/policy.cc

```diff
@@ -0,0 +1,155 @@
+#include "policy.h"
+#include "base_object-inl.h"
+#include "env-inl.h"
+#include "memory_tracker-inl.h"
+#include "node.h"
+#include "node_external_reference.h"
+#include "node_errors.h"
+
+#include "v8.h"
+
+#include <string>
+#include <vector>
+#include <iostream>
+
+namespace node {
+
+using v8::Array;
+using v8::Context;
+using v8::FunctionCallbackInfo;
+using v8::Integer;
+using v8::Just;
+using v8::Local;
+using v8::Maybe;
+using v8::Nothing;
+using v8::Object;
+using v8::String;
+using v8::Value;
+
+namespace policy {
+
+// The root policy is establish at process start using
+// the --policy-deny-* command line arguments.
+Policy root_policy;
+
+namespace {
+
+// policy.deny('fs.in', ['/tmp/'])
+// policy.deny('fs.in')
+static void Deny(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  v8::Isolate* isolate = env->isolate();
+
+  CHECK(args[0]->IsString());
+  CHECK(args.Length() >= 2 || args[1]->IsArray());
+
+  std::string denyScope = *String::Utf8Value(isolate, args[0]);
```

Comment:
https://github.com/nodejs/node/blob/main/doc/contributing/cpp-style-guide.md#snake_case-for-local-variables-and-parameters